test/SubjectPathResolver-test.js

```diff
@@ -0,0 +1,35 @@
+import { PathFactory } from 'ldflex';
+import SubjectPathResolver from '../src/SubjectPathResolver';
+import ComunicaUpdateEngine from '../src/ComunicaUpdateEngine';
+
+jest.mock('ldflex');
+
+describe('a SourcePathSubjectResolver', () => {
+  let resolver;
+  const source = { dummy: 'source' };
+
+  beforeEach(() => {
+    resolver = new SubjectPathResolver(new PathFactory(), source);
+  });
+
+  it('supports strings', () => {
+    expect(resolver.supports('apple')).toBeTruthy();
+  });
+
+  it('does not support non-strings', () => {
```

Comment:
You might want to change this into `does not support symbols` and test with `new Symbol()`, since only strings and symbols will ever occur as arguments.